core/platform/taskqueue/gae_taskqueue_services.py

```diff
@@ -36,6 +38,13 @@ def defer_to_events_queue(fn, *args, **kwargs):
     # details on the _queue kwarg.
     deferred.defer(fn, *args, _queue=QUEUE_NAME_EVENTS, **kwargs)
 
+def add_feedback_message_email_task(url, **kwargs):
```

Comment:
This still knows too much about the domain layer. How about something generic like enqueue_task()?

I think, also, let's maybe do this with explicit args (so specify 'params', 'countdown', etc. and what they mean explicitly in the method signature). If we want to modify the platform layer in the future this will make it easier to reason about than if we just have a **kwargs blob.